llvm/test/tools/llvm-cxxfilt/quote.test

```diff
@@ -1,12 +1,21 @@
-// Show that llvm-cxxfilt --quote can emit quoted demangled symbols but not
-// without double-quoting in the case they are already quoted
+// Show that llvm-cxxfilt --quote can emit quoted demangled symbols (even at the
+// end of a line) but not without double-quoting in the case they are already
+// quoted.
```

Comment:
The phrasing here is a bit of a mouthful. How about this suggestion.
```suggestion
// Show that llvm-cxxfilt --quote adds quotes around demangled symbols, unless the symbol is already quoted.
```
(with appropriate line wrapping)